### PR TITLE
Replace snippet with complete code

### PR DESCRIPTION
**Pull request by amarantha-k**

### CODE REVIEW

**Review by maria-robobug — APPROVED** [review]
👍 LGTM

File: modules/hello-world/pages/start-using-sdk.adoc [hunk 1]
```diff
@@ -114,14 +114,7 @@ Local Couchbase Server::
...
 ----
-include::example$StartUsing.java[tag=connect,indent=0]
-
-include::example$StartUsing.java[tag=bucket,indent=0]
-
-include::example$StartUsing.java[tag=collection,indent=0]
-
-include::example$StartUsing.java[tag=upsert,indent=0]
-
+include::example$StartUsing.java
```
Comment (maria-robobug) [inline modules/hello-world/pages/start-using-sdk.adoc]: oh wait I think you may need `[]` at the end to see the full example.

Comment (osfameron) [reply modules/hello-world/pages/start-using-sdk.adoc]: and `[tags=**]` will strip the antora tag declarations

Comment (maria-robobug) [reply modules/hello-world/pages/start-using-sdk.adoc]: yep ^ that should do it

**Pull request merged**